Xamarin.Forms.Platform.iOS/Renderers/NavigationRenderer.cs

```diff
@@ -320,16 +320,16 @@ protected virtual void OnElementChanged(VisualElementChangedEventArgs e)
 
 			if (poppedViewController == null)
 			{
-				// this happens only when the user does something REALLY dumb like pop right after putting the page as visible.
-				poppedViewController = TopViewController;
-				var newControllers = ViewControllers.Remove(poppedViewController);
-				ViewControllers = newControllers;
+				//// this happens only when the user does something REALLY dumb like pop right after putting the page as visible.
```

Comment:
Did you mean to leave these changes in?

Either way, we should remove the "really dumb" from this comment.